docs/user/player/advanced/auto-login.md

```diff
@@ -0,0 +1,61 @@
+# Auto Login
+
+:::warning
+
+This feature requires a good understanding of Graasp and might be for the more technically inclined.
+
+:::
+
+In this article we will use the term `itemLogin` to designate the possibility to log a user with a pseudonym.
+
+## Purpose {#purpose}
+
+This feature allows to force the itemLogin account used for a lesson by providing a url that contains all necessary information to the user.
+
+##  Use cases {#use-cases}
+
+Potential use-cases for this feature include:
+
+- Studies (like Prolific) where you can provide a personalized link to each participant and you **need** each participant to use a very specific username (for data analysis purposes for example)
```

Comment:
"you need" but "maybe for example" for analysis purpose. Do you need it or not? Maybe "you want" fits better.